### ISSUE TITLE
Custom Images Fails 

### ISSUE THREAD

**Issue by dev-yeet** [google/restor#37]
If I compile my own project using our team project ID, it starts to work fine (I've gotten to where it deletes the partition), but then it will fail and tell me that I have to use 10.13+ to restore an APFS image. I'm running 10.13.15 B3, so that's clearly not the case. If i use an HFS image, I get " Could not unmount volume "/Volumes/%noformat" resource busy. 

Using the release dmg doesn't give me either of these errors but will just hang indefinitely on "starting". I'm not really sure what to do, and i've poked at this for about 2 days now. Any insight into either of these issues would be much appreciated.

**Comment by tburgin** [comment]
When doing local development you will need to remove the cached `com.google.corp.restord` binary before your changes will take effect.

`sudo rm /Library/PrivilegedHelperTools/com.google.corp.restord`

I know this is not very clear, we can do a better job to surface this. Or just find a way to fix it ;)

**Comment by dev-yeet** [comment]
hi thanks for the quick response. I tried deleting the file, but it comes back when running the program requires my admin privileges and I still get the error for both parts on the app I built locally. 

Edit: I've made progress, however I'm now stumped on the APFS issued referenced  in issue #27.  Imaging HFS works if i use 1.5/1.6 release, but APFS won't bless. This didn't happen before on my machine running 10.13.5. As it should have worked on 10.13.4, something we've put in our images must be interfering because both 1.5 and 1.6 won't bless the partition, and I get "The container for this APFS volume doesn't appear to have a preboot volume." I'll close this issue since it must be specific to my machine or environment.
